src/test/kotlin/no/nav/helse/flex/FellesTestOppsett.kt

```diff
@@ -200,6 +201,17 @@ abstract class FellesTestOppsett : TestOppsettInterfaces {
         arbeidssokerregisterStoppConsumer.hentProduserteRecords().shouldBeEmpty()
     }
 
+    @AfterAll
+    fun `Vi sjekker om det er noen responser igjen i rest service serverne`() {
+        val mockViSjekker = listOf(AaregMockDispatcher, BrregMockDispatcher)
+        mockViSjekker.forEach {
```

Comment:
Mulig feilmeldingen blir "bedre" (aka slipper Stack-trace fra exception) ved å bruke en Kluent assert type it.harRequestsIgjen() `should be equal to` false el.lign?